src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/client/model/offender/HighLight.kt

```diff
@@ -0,0 +1,15 @@
+package uk.gov.justice.digital.hmpps.personrecord.client.model.offender
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.annotation.JsonProperty
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+data class HighLight(
```

Comment:
The case is a bit confusing - Highlight might be better but I suppose this is how the JSON is

Reply:
Probably a typo , will check and change.